brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-827 Testimonial block default mobile view - Arrow placement issue fixed

### Description
The arrows to navigate through testimonials overlaps the testimonial itself.

### Screenshots
Issue - https://d.pr/draw/e6youR/
Fixed - https://share.getcloudapp.com/z8umqEKO

### Types of changes

Bug fix (non-breaking change which fixes an issue)



### How has this been tested?

- Add single container
- Add testimonial block
- Publish the page
- Check the page in mobile view

### Checklist:
